macros/netatalk.m4

```diff
@@ -113,11 +113,15 @@ AC_DEFUN([AC_NETATALK_DBUS_GLIB], [
     AC_MSG_ERROR([afpstats requested but dbus-glib not found])
   fi
 
+  if test x$prefix = xNONE ; then
+    prefix=/usr/local
+  fi
```

Comment:
Are there known states when $prefix resolves to NONE still that we need to hedge against?